Resources/Catalog/FileCatalogProxyClient.py

```diff
@@ -31,6 +32,12 @@ def getName( self ):
     """
     return self.fcName
 
+  def setInterfaceMethods( self, methodTuple ):
+    self.interfaceMethods = methodTuple
+
+  def getInterfaceMethods( self ):
+    return self.interfaceMethods
+
```

Comment:
Well, fine, this is a bit religious but OK. Here, the class should implement getInterfaceMethods() as it is part of its base interface like for other FC classes where this is not just a getter method.